pkg/controller/operands/cdi.go

```diff
@@ -159,6 +159,24 @@ func NewCDI(hc *hcov1beta1.HyperConverged, opts ...string) (*cdiv1beta1.CDI, err
 		hc.Spec.Workloads.NodePlacement.DeepCopyInto(&spec.Workloads)
 	}
 
+	if hc.Spec.CertConfig != nil {
+		spec.CertConfig = &cdiv1beta1.CDICertConfig{}
+
+		if hc.Spec.CertConfig.CA != nil {
+			spec.CertConfig.CA = &cdiv1beta1.CertConfig{
+				Duration:    hc.Spec.CertConfig.CA.Duration.DeepCopy(),
```

Comment:
can't we simply copy the whole struct?

Reply:
ok, we don't have a common and shared struct in controller-lifecycle-operator-sdk
sorry for the noise